### ISSUE TITLE
Prettify code

### ISSUE THREAD

**Issue by Gabriel-Alves-Cunha** [jctaoo/electron-run#53]
Hey, I was looking to update prettier to latest, and when it does, it needs to prettify a lot of files. I was thinking into maybe baking it into github actions, like this:

```yml
# File ".github/workflows/CI.yml"
# [...]
      - name: install dependencies
        if: steps.cache.outputs.cache-hit != 'true'
        run: |
          yarn install
          yarn global add npm-publish

# New code:
      - name: Prettify code
        run: |
          echo "Prettifying code"
          yarn fix:prettier --loglevel silent
# End of new code.

      - name: install dependencies for fixtures
        run: |
          cd fixtures/demo
          yarn install
          cd ../../
# [...]
```
What are your thoughts?

**Comment by jctaoo** [comment]
You mean check code whether is handled by prettier in GitHub actions? T🤔

**Comment by Gabriel-Alves-Cunha** [comment]
I mean: make GA handle the prettifying of code; like, so you don't have to manually review all of the changed files prettified in a pull request.

By changing that .yml file above, prettier would run on the server side, prettifying all the code before 'pull request-ing' into main; also running before 'yarn test', because 'yarn test' checks if all code is pretty. 
I read about it, among others, here: https://dev-yakuza.posstree.com/en/react/github-actions/prettier-eslint/

**Comment by jctaoo** [comment]
It looks good!

**Comment by jctaoo** [comment]
closed by #56
